client/me/purchases/payment-methods/payment-method-delete-dialog.tsx

```diff
@@ -44,8 +62,53 @@ const PaymentMethodDeleteDialog: FunctionComponent< Props > = ( {
 					}
 				) }
 			</p>
+			{ associatedSubscriptions.length > 0 && (
+				<div className="payment-method-delete-dialog__affected-subscriptions-wrapper">
+					<div className="payment-method-delete-dialog__affected-subscriptions-title-wrapper">
+						<CardHeading tagName="h2" size={ 20 }>
+							{ translate( 'Associated subscriptions' ) }
+						</CardHeading>
+						<img src={ getPaymentMethodImageURL( card?.card_type ) } alt="" />
+					</div>
+					{ associatedSubscriptions.map( ( subscription: Purchase ) => (
+						<div
+							className="payment-method-delete-dialog__affected-subscription"
+							key={ subscription.id }
+						>
+							<div>
+								<span>{ subscription.productName }</span>
+								<span className="payment-method-delete-dialog__affected-subscription-domain">
+									{ subscription.meta || subscription.domain }
+								</span>
+							</div>
+							<div>
+								<span className="payment-method-delete-dialog__affected-subscription-date">
+									{ translate( 'Auto-renews{{br /}}%(date)s', {
```

Comment:
I know the code in calypso does this a lot already, but I wonder if there's a way to style this text without using a `br` tag and instead relying on CSS. 🤔 The reason is that because the tag is in the localized string, any time we want to change its layout, we have to change the string and trigger re-translations. If the newline were part of the markup or the CSS, then we could change how it looks without involving translators at all.

Reply:
I think so, but I'm not sure if this can be done while still keeping the date in the string (let me know if I'm missing something obvious though!). I've added a comment to the string instead - let me know if that approach works.